ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTI0OTQ4Mjg1LjE3NDYzMTQzNzQifSwiYW5vbklkIjoiMDY4ZTIyNTNiOTc1YWEwZiJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5sdS9mci9wYXNzZW5nZXJjYXJzL21vZGVscy9lc3RhdGUvYy1jbGFzcy9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI1NCIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjI0Njk3NTI5NzQwMDA4NTk1IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x5568dc33575a <unknown>
#1 0x5568dbde84b0 <unknown>
#2 0x5568dbdced86 <unknown>
#3 0x5568dbdcd69a <unknown>
#4 0x5568dbdcddff <unknown>
#5 0x5568dbdcdd54 <unknown>
#6 0x5568dbdec69f <unknown>
#7 0x5568dbe9192a <unknown>
#8 0x5568dbe5f592 <unknown>
#9 0x5568dbe855e0 <unknown>
#10 0x5568dbe5f363 <unknown>
#11 0x5568dbe2bd63 <unknown>
#12 0x5568dbe2c9c1 <unknown>
#13 0x5568dc2faa6b <unknown>
#14 0x5568dc2fe951 <unknown>
#15 0x5568dc2e2b62 <unknown>
#16 0x5568dc2ff4c4 <unknown>
#17 0x5568dc2c713f <unknown>
#18 0x5568dc3236f8 <unknown>
#19 0x5568dc3238d6 <unknown>
#20 0x5568dc3345a6 <unknown>
#21 0x7f286c89caa4 <unknown>
#22 0x7f286c929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjEyNDk0ODI4NS4xNzQ2MzE0Mzc0In0sImFub25JZCI6IjA2OGUyMjUzYjk3NWFhMGYifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnoubHUvZnIvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvZXN0YXRlL2MtY2xhc3Mvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNTQiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjoyfSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiIxMjA2MjY2MDc0OTU2NDY0NyIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x5568dc33575a <unknown>
#1 0x5568dbde84b0 <unknown>
#2 0x5568dbdced86 <unknown>
#3 0x5568dbdcd69a <unknown>
#4 0x5568dbdcddff <unknown>
#5 0x5568dbdcdd54 <unknown>
#6 0x5568dbdec69f <unknown>
#7 0x5568dbe9192a <unknown>
#8 0x5568dbe5f592 <unknown>
#9 0x5568dbe855e0 <unknown>
#10 0x5568dbe5f363 <unknown>
#11 0x5568dbe2bd63 <unknown>
#12 0x5568dbe2c9c1 <unknown>
#13 0x5568dc2faa6b <unknown>
#14 0x5568dc2fe951 <unknown>
#15 0x5568dc2e2b62 <unknown>
#16 0x5568dc2ff4c4 <unknown>
#17 0x5568dc2c713f <unknown>
#18 0x5568dc3236f8 <unknown>
#19 0x5568dc3238d6 <unknown>
#20 0x5568dc3345a6 <unknown>
#21 0x7f286c89caa4 <unknown>
#22 0x7f286c929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyMDYyIiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJmcl9MVSIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lmx1L2ZyL3Bhc3NlbmdlcmNhcnMvbW9kZWxzL2VzdGF0ZS9jLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjU0In0sImZsYWdzIjp7InBhZ2VWaWV3Ijp0cnVlfSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxMjQ5NDgyODUuMTc0NjMxNDM3NCJ9LCJhbm9uSWQiOiIwNjhlMjI1M2I5NzVhYTBmIn0sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6Mn0sImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX0sImNhdGFsb2ciOnsiUHJvZHVjdCI6eyJfaWQiOiIyMDYyIiwiZGltZW5zaW9ucyI6eyJCYXVtdXN0ZXJTaG9ydCI6WyIyMDYyIl19fX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiNDQ4MzE1Njc0NjI3NTIyNCJ9: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x5568dc33575a <unknown>
#1 0x5568dbde84b0 <unknown>
#2 0x5568dbdced86 <unknown>
#3 0x5568dbdcd69a <unknown>
#4 0x5568dbdcddff <unknown>
#5 0x5568dbdcdd54 <unknown>
#6 0x5568dbdec69f <unknown>
#7 0x5568dbe9192a <unknown>
#8 0x5568dbe5f592 <unknown>
#9 0x5568dbe855e0 <unknown>
#10 0x5568dbe5f363 <unknown>
#11 0x5568dbe2bd63 <unknown>
#12 0x5568dbe2c9c1 <unknown>
#13 0x5568dc2faa6b <unknown>
#14 0x5568dc2fe951 <unknown>
#15 0x5568dc2e2b62 <unknown>
#16 0x5568dc2ff4c4 <unknown>
#17 0x5568dc2c713f <unknown>
#18 0x5568dc3236f8 <unknown>
#19 0x5568dc3238d6 <unknown>
#20 0x5568dc3345a6 <unknown>
#21 0x7f286c89caa4 <unknown>
#22 0x7f286c929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFBpUGFnZSI6eyJ2ZWhpY2xlUEkiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lmx1L2ZyL3Bhc3NlbmdlcmNhcnMvbW9kZWxzL2VzdGF0ZS9jLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ2ZWhpY2xlQ29kZSI6InMyMDYiLCJ2ZWhpY2xlQk0iOiIyMDYyIiwidmVoaWNsZVBJSGlzdG9yeSI6IjIwNjIiLCJ0aW1lc3RhbXAiOjE3NDYzMTQzODAwNDIsImV4cGlyYXRpb24iOjEyMH19fSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiIxMjQ5NDgyODUuMTc0NjMxNDM3NCJ9LCJhbm9uSWQiOiIwNjhlMjI1M2I5NzVhYTBmIn0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lmx1L2ZyL3Bhc3NlbmdlcmNhcnMvbW9kZWxzL2VzdGF0ZS9jLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjU0IiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI5Mzg1NDk5NTI3Mzk4NTgxIiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x5568dc33575a <unknown>
#1 0x5568dbde84b0 <unknown>
#2 0x5568dbdced86 <unknown>
#3 0x5568dbdcd69a <unknown>
#4 0x5568dbdcddff <unknown>
#5 0x5568dbdcdd54 <unknown>
#6 0x5568dbdec69f <unknown>
#7 0x5568dbe9192a <unknown>
#8 0x5568dbe5f592 <unknown>
#9 0x5568dbe855e0 <unknown>
#10 0x5568dbe5f363 <unknown>
#11 0x5568dbe2bd63 <unknown>
#12 0x5568dbe2c9c1 <unknown>
#13 0x5568dc2faa6b <unknown>
#14 0x5568dc2fe951 <unknown>
#15 0x5568dc2e2b62 <unknown>
#16 0x5568dc2ff4c4 <unknown>
#17 0x5568dc2c713f <unknown>
#18 0x5568dc3236f8 <unknown>
#19 0x5568dc3238d6 <unknown>
#20 0x5568dc3345a6 <unknown>
#21 0x7f286c89caa4 <unknown>
#22 0x7f286c929c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Allure-Testing/QA-Allure-Testing/Tests/BFV1_attempt_2.png'